api/python/cellxgene_census/release_process.md

```diff
@@ -81,25 +81,37 @@ To create a release, perform the following:
    ```
 4. When the workflow completes, make note of the run ID (e.g., using `gh run list`).
 5. Optional, _but recommended_: download the asset from the build workflow and validate it.
-6. Create and publish a GitHub Release, using the `<SEMVER>` tag above (e.g., `v1.9.4`). It is recommended that you include a release summary and change log in the GH release.
+6. Create and publish a GitHub Release [here](https://github.com/chanzuckerberg/cellxgene-census/releases/new). Set the release title to the `<SEMVER>`. Use the `Generate Release Notes` button to auto-populate the summary with a changelog. It is reasonable to remove any R-specific or builder-specific entries.
 
 ## Step 4: Publish assets to PyPi
 
 To publish built release assets to PyPi (_note_: this will require your pypi/testpypi login):
 
-1. Download the assets built for your release commit, using the same method as step 2 above, e.g.,
+1. Delete any existing release builds you may have accumulated in the past: `rm ./artifact/*`.
+2. Download the assets built for your release commit, using the same method as step 2 above, e.g.,
    ```shell
    $ gh run download <ID>
    ```
-2. Optional: upload to TestPyPi (this assumes the downloaded assets are in ./artifact/).
+3. Optional: upload to TestPyPi (this assumes the downloaded assets are in ./artifact/).
 
    ```shell
    pipx run twine upload --repository testpypi ./artifact/*
    ```
 
-   Following the upload, confirm correct presentation on the project page and ability to download install from TestPyPi. To test with TestPyPi, use `pipx run twine upload --repository testpypi ./artifact/*`. You can find more information [here](https://packaging.python.org/en/latest/guides/using-testpypi/).
-
-3. Use twine to upload to PyPi (this assumes the downloaded assets are in ./artifact/), e.g.,
+   Following the upload, confirm correct presentation on the project page and ability to download install from TestPyPi. 
+4. To test installation from TestPyPi:
+   ```shell
+   pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple cellxgene-census
+   python -c "import cellxgene_census; print(cellxgene_census.__version__)"
+   ```
+   Note that the `--extra-index-url` option ensures that any transitive package dependencies that are _not_ available on
+   `test.pypi.org` can be satisfied by installing them from the production `pypi.org`.  You can find more information
+   [here](https://packaging.python.org/en/latest/guides/using-testpypi/).
+5. Use twine to upload to PyPi (this assumes the downloaded assets are in ./artifact/), e.g.,
    ```shell
    pipx run twine upload ./artifact/*
+6. Test the installation from PyPi, as a final sanity check. Note that it may take a minute for the new release to be visible on pypi.org:
+   ```shell
+   pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple cellxgene-census
```

Comment:
this looks like a typo, e.g., "test.pypi.org"

Reply:
woops, "copy & paste w/o edit" error